### DISCUSSION TITLE
[Question][HW2] I am using scapy to send packets. However, victim does not receive packets

### DISCUSSION THREAD

**Discussion by mrpositron** [prosyslab-classroom/cs348-information-security#501] [Q&A]
When I set current IP address as a source IP in the packet, and then send it. I get response from the AMP. However, when I change source IP to the victim's IP, the packet does not reach VICTIM. I use tcpdump to see if packets arrive. 

Could you help me?

**Comment by Altterisk** [comment]
Not a fix since I don't use Scapy and have no idea what could have gone wrong with it, but you should try using Wireshark instead of tcpdump. 
I have cases where tcpdump couldn't catch packets but Wireshark can (just tested now, Wireshark got 60+ packets from my script but tcpdump got none). Plus Wireshark can help you better understand the structure of each header/layer to construct better packets.
